src/clients/ProfitClient.ts

```diff
@@ -153,20 +153,41 @@ export class ProfitClient {
     this.unprofitableFills = {};
   }
 
+  // Allow the minimum relayer fee to be overridden per token/route:
+  // 0.1bps on USDC from Optimism to Arbitrum:
+  //   - MIN_RELAYER_FEE_PCT_USDC_42161_10=0.00001
+  minRelayerFeePct(symbol: string, srcChainId: number, dstChainId: number): BigNumber {
+    const routeKey = `${symbol}_${srcChainId}_${dstChainId}`;
+    let minRelayerFeePct = this.minRelayerFees[routeKey];
+
+    if (!minRelayerFeePct) {
+      const _minRelayerFeePct = process.env[`MIN_RELAYER_FEE_PCT_${routeKey}`];
+      minRelayerFeePct = _minRelayerFeePct ? toBNWei(_minRelayerFeePct) : this.defaultMinRelayerFeePct;
+
+      // Save the route for next time.
+      this.minRelayerFees[routeKey] = minRelayerFeePct;
+    }
+
+    return minRelayerFeePct as BigNumber;
```

Comment:
Optional suggestion to streamline this flow a little.

```suggestion
    if (!this.minRelayerFees[routeKey]) {
      const _minRelayerFeePct = process.env[`MIN_RELAYER_FEE_PCT_${routeKey}`];
      this.minRelayerFees[routeKey] = _minRelayerFeePct ? toBNWei(_minRelayerFeePct) : this.defaultMinRelayerFeePct;
    }
    
    return this.minRelayerFees[routeKey];
```